reana_server/reana_admin.py

```diff
@@ -524,25 +526,50 @@ def list_quota_resources(ctx, admin_access_token):
         "E.g. --email johndoe@example.org --email janedoe@example.org"
     ),
 )
-@click.option("--resource-name", "-r", help="Name of resource.", required=True)
+@click.option(
+    "--resource", "-r", help="Specify quota resource. e.g. cpu, disk.", required=True
+)
```

Comment:
Should this be required? If one wants to set a limit to a single resource name, how should they proceed?

Reply:
I think yes, if one wants to set a limit to a resource:

- on one-type-one-resource scenario:
```kubectl exec -i -t deployment/reana-server -- flask reana-admin quota-set -r disk ...```
- on one-type-multi-resource scenario one has to additionally specify resource name:
```kubectl exec -i -t deployment/reana-server -- flask reana-admin quota-set -r disk --resource-name "my disk" ...```

Reply:
> - on one-type-multi-resource scenario one has to additionally specify resource name:
```kubectl exec -i -t deployment/reana-server -- flask reana-admin quota-set -r disk --resource-name "my disk" ...```


The resource name is unique, so requiring to pass also the resource type is redundant.

Reply:
Agree that it's redundant. My idea was that on one-type-one-resource scenario user would see `resource` as a required field while doing `reana-admin quota-set --help` and it would signal that this is the option one has to use.. (since we should focus on this use case).
Also, supporting `resource-name` without `resource` uglifies the implementation a bit